models/release.go

```diff
@@ -217,6 +223,28 @@ func GetReleasesByRepoID(repoID int64, opts FindReleasesOptions) ([]*Release, er
 	return rels, sess.Find(&rels)
 }
 
+// GetLatestReleaseByRepoID returns the latest release for a repository
+func GetLatestReleaseByRepoID(repoID int64) (*Release, error) {
+	cond := builder.NewCond().
+		And(builder.Eq{"repo_id": repoID}).
+		And(builder.Eq{"is_draft": false}).
+		And(builder.Eq{"is_prerelease": false}).
+		And(builder.Eq{"is_tag": false})
+
+	rel := new(Release)
+	has, err := x.
+		Desc("created_unix", "id").
+		Where(cond).
+		Get(rel)
```

Comment:
I'm not sure, but I believe xorm will return an error if the query results have a record count > 1.

Reply:
dont think so

Reply:
@guillep2k -> https://gitea.com/xorm/xorm/src/branch/master/README.md

`Get query one record from database`

Reply:
I've checked this in xorm code and it turns out you're right. It worries me because I've recently suggested a change in another PR that depended on this inexisting behaviour. 😱 (it was a consistency check for defensive programming, so there should be no consequences, but still). 😳